docs/source/standard/auction.rst

```diff
@@ -219,4 +235,36 @@ Bank Account
 
   * `UA-EDR`; 
   * `UA-MFO`;
-  * `accountNumber`.
\ No newline at end of file
+  * `accountNumber`.
+
+.. _contracts:
+
+Сontracts
+=========
+
+Schema
+------
+
+:type:
+  string, required
+
+  contract type. The only meaning is `yoke`.
+
+:id:
+  string, required
+
+  internal ID of the object that is created in array.
+
+:contractID:
+  string, required
+
+  Added after the contract is created.
+
+  External contract ID.
```

Comment:
The contract identifier to refer to in “paper” documentation.

Added as long as the contract is being created within the Module of Contracting.